server/firestore.rules

```diff
@@ -0,0 +1,10 @@
+rules_version = '2';
```

Comment:
Isn't this version #1 for this project?
```suggestion
rules_version = '1';
```

Reply:
Ah, the `rules_version` refers to the version of the _Firebase Rules language_, it's not our own version number.